### ISSUE TITLE
Typo: Manual Config

### ISSUE THREAD

**Issue by sasugajay** [CrackyStudio/arkso#13]
Rendered completely unusable.  "Maunal Config" on main menu is clearly a typo of Manual.

**Comment by CrackyStudio** [comment]
Yes, resolution is already planned in [v5.2](https://github.com/CrackyStudio/arkso/projects/3)
